tenants/multi/config/core.js

```diff
@@ -74,4 +74,19 @@ module.exports = {
       },
     ],
   },
+
+  mdd: {
+    socialMediaLinks: [
+      {
+        provider: 'facebook',
+        href: 'https://www.facebook.com/medicaldesigndevelopment/',
+        title: 'Follow us on Facebook',
+      },
+      {
+        provider: 'twitter',
+        href: '#https://twitter.com/MedDevelopment',
```

Comment:
Is this suppose to have a # in front of it?

Reply:
@Shinsina Nope! Missed that in my deletion.